frontend/src/components/SignUp/SignUp.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Box from '../common/Box/Box';
+import Button from '../common/Button/Button';
+
+const SignUpBlock = styled.div``;
+
+SignUp.propTypes = {
+  form: PropTypes.object.isRequired,
+  onChange: PropTypes.func.isRequired,
+  onClick: PropTypes.func.isRequired,
+};
+
+function SignUp({ form, onChange, onClick }) {
+  return (
+    <SignUpBlock>
+      <h3>Join Ya-Ta!</h3>
```

Comment:
please change all heading tag `<h1>`, `<h2>`, ... into `<Typography variant="h1">` ... 

you need to `import { Typography } from '@material-ui/core';`